app/scripts/modules/core/widgets/accountRegionClusterSelector.component.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular
+  .module('spinnaker.core.accountRegionClusterSelector.directive', [
+    require('../../core/application/listExtractor/listExtractor.service'),
+  ])
+  .directive('accountRegionClusterSelector', function() {
```

Comment:
I see the file is named `accountRegionClusterSelector.component.js` - did you want to write this as a `component`?